extensions/mssql/package.json

```diff
@@ -837,6 +837,7 @@
           "groupName": "%mssql.connectionOptions.groupName.source%",
           "valueType": "string",
           "defaultValue": null,
+          "placeHolder": "e.g. tcp:servername,1433",
```

Comment:
For (2):

Not using `e.g.` might imply this is default value, as placeholder for 'Database' also sets `<Default>` as placeholder which corresponds to actual default value. 
![image](https://user-images.githubusercontent.com/13396919/231550802-77222fff-ab83-4fff-b63d-5c75f53017a8.png)
To avoid confusion, I went with `e.g.` as that's understandable as an example, and that is our purpose.

I'd be adding descriptions for all main dialog options together, as we currently don't have any description what-so-ever for any of the main connection dialog options, except the newly added Encryption options.. and that's a feedback we've got internally we should be addressing. It will also require other providers to update their descriptions to user friendly information, so we'll follow up on it separately.